compiler/packages/snap/src/compiler.ts

```diff
@@ -124,6 +125,9 @@ function makePluginOptions(
   if (firstLine.includes("@enablePreserveExistingManualUseMemo")) {
     enablePreserveExistingManualUseMemo = true;
   }
+  if (firstLine.includes("@disableMemoizationForDebugging")) {
+    disableMemoizationForDebugging = true;
+  }
```

Comment:
same as previous PR, this should get picked up by the call to parseConfigPragmaFn